pdns/recursordist/docs/settings.rst

```diff
@@ -1523,6 +1523,15 @@ IP address for the webserver to listen on.
 
 These subnets are allowed to access the webserver.
 
+.. _setting-webserver-max-bodysize:
+
+``webserver-max-bodysize``
+--------------------------
+-  Integer
```

Comment:
```suggestion
.. versionadded:: 4.2.0

-  Integer
```

Reply:
Oh yes, sorry. Will hold off on this a bit in case we also backport.

Reply:
committed